cluster-management/README.md

```diff
@@ -1,13 +1,28 @@
-## Cluster Management
+# Cluster Management
 
 This directory contains scripts specific to cluster managment intended to be run after an OCP installation for operationalization purposes.
 
-### Contents:
+## Cluster Capacity Report (cluster_capacity.py)
 
-1. Garbage collection
-- Sets up spotify docker-gc on hosts. Should be run a standard 'bring your own' OpenShift-ansible inventory file. The playbook will set up garbage collection on all host with the [node] designation. 
-- View the documentation for spotify docker-gc here: [LINK](https://github.com/spotify/docker-gc)
+The `cluster_capacity.py` script generates a report showing how full a kubernetes cluster is. It can be run from anywhere there is an active `~/.kube/config` context and requires at least `cluster-reader` or equivalent permissions.
```

Comment:
Can you mention that this script is in the `capacity` folder. or add a `cd capacity` instruction in the `README.md`

Reply:
Done